<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>525</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>525</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>525</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>500</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.483, y1:0, x2:0.472637, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(0, 40, 62, 255));
border-radius:20px;
border:1px solid #00007f;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>491</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0,0%);
font: 75 25pt &quot;Arial&quot;;
border:None</string>
       </property>
       <property name="text">
        <string>Detector Aviones Militares</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>130</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0,0%);
font: 75 15pt &quot;Arial&quot;;
border:None</string>
       </property>
       <property name="text">
        <string>Usuario</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>230</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0,0%);
font: 75 15pt &quot;Arial&quot;;
border:None;</string>
       </property>
       <property name="text">
        <string>Contraseña</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="users">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>160</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px;
border:None</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Ingrese su usuario</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="password">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>260</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px;
border:None</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Ingrese su contraseña</string>
       </property>
      </widget>
      <widget class="QPushButton" name="bt_ingresar">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>340</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(0, 255, 0);
	font: 75 12pt &quot;MS Shell Dlg 2&quot;
}
QPushButton:hover{
	background-color: rgb(255, 255, 0);
	font: 75 12pt &quot;MS Shell Dlg 2&quot;
}</string>
       </property>
       <property name="text">
        <string>Iniciar Sesión</string>
       </property>
      </widget>
      <widget class="QPushButton" name="salir">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>440</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgba(0, 0, 0,0%);
	font: 75 12pt &quot;MS Shell Dlg 2&quot;;
	border:None;
}
QPushButton:hover{
	background-color: rgb(255, 255, 0);
	font: 75 12pt &quot;MS Shell Dlg 2&quot;
}</string>
       </property>
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
      <widget class="QLabel" name="usuario_incorrecto">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>190</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0,0%);
font: 75 10pt &quot;Arial&quot;;
color:rgb(255, 0, 0);
border:None;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="contrasena_incorrecta">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>290</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0,0%);
font: 75 10pt &quot;Arial&quot;;
color:rgb(255, 0, 0);
border:None;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="cargando">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>380</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0,0%);
font: 75 10pt &quot;Arial&quot;;
color:rgb(255, 0, 0);
border:None;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>salir</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>552</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
